### ISSUE TITLE
Posting to YouMagine failed 3 times loosing long text and image selection

### ISSUE THREAD

**Issue by garycmartin** [daid/LegacyCura#732]
Tl:dr better sanity checking / error messages needed on Cura YM upload process, and it should not just discard all user input if it does fail for some reason.

I had a simple vase mesh to upload (my first upload). After typing a good couple of paragraphs of detailed text and adding images, hitting post closed the YM input window and presented a 'failed to create dialogue', loosing all the text and selection of images. I tried again and with the same result, third time I remembered to copy paste the text so as not to lose it. It failed again. Went to the Youmagine web site and finally posted it from there successfully. The bug appears to be that the Youmagine didn't like receiving a handful of 2592x3516 (2Mb) jpg photos (the web page form only accepted a couple of them before failing to upload any more). After downsizing the images the web page accepted all of them and published the post fine.

**Comment by daid** [comment]
Apperently, YouMagine put a 300 character limit on the first description field. Working on handling this better in Cura right now.
